client/components/vote-escrow/LockupForm.tsx

```diff
@@ -188,8 +188,8 @@ const LockupForm: FunctionComponent<LockupFormProps> = ({ existingLockup }) => {
       transaction = await contracts.OriginDollarGovernance.approve(
         contracts.OgvStaking.address,
         ethers.constants.MaxUint256,
-        { gasLimit: 1000000 }
-      ); // @TODO maybe set this to lower
+        { gasLimit: 66983 }
```

Comment:
That looks too tight. We did an approve from the multisig and it used 149,604
https://etherscan.io/tx/0x3f0ba0620459079f8b2d55c380f2409aa876e8776c2b481281af09061ebebb3a

Reply:
Wrong hash. Here is the one:
https://etherscan.io/tx/0x059faaef43ca156e2da0624bf63a04cd3cb841f1da11c6772f70dd0577b38b17
 
Using 111,065